src/detect-tls-version.c

```diff
@@ -185,14 +187,17 @@ static DetectTlsVersionData *DetectTlsVersionParse (const char *str)
             tmp_str += 1;
         }
 
-        if (strcmp("1.0", tmp_str) == 0) {
+        if (strncmp("1.0", tmp_str, 3) == 0) {
```

Comment:
strNcmp seems wrong here. This would match 1.0.1 as well.

Reply:
Same question here as above :)